myoolala/react-starter
# Feat/add e2e encryption

## Acceptance Criteria
Update the app/terraform to support encrypted communication between the load balancer and the container

## Change Description
Adds the final piece for e2e encryption

## Verification Instructions
Deploy the app via terraform in ssl mode and verify the containers pass ALB healthchecks

## Commit Message
feat: adds e2e encryption between the ALB and container
